src/bloaty.cc

```diff
@@ -781,6 +817,19 @@ void RollupOutput::PrettyPrint(const OutputOptions& options,
 
   // The "TOTAL" row comes after all other rows.
   PrettyPrintRow(toplevel_row_, 0, options, out);
+
+  uint64_t filtered = 0;
+  if (ShowFile(options)) {
+    filtered += toplevel_row_.filtered_filesize;
```

Comment:
I don't think adding the two together is the right approach. Your existing format is good for the case that we're showing only VM or file:

```
Filtering enabled (source_filter); omitted 15.1Mi of entries
```

For the case that both VM and file are shown lets do this instead:

```
Filtering enabled (source_filter); omitted file=12.5Mi, vm=1.2Mi of entries
```

Reply:
That makes sense, updated.